slices/ca-certificates.yaml

```diff
@@ -16,6 +16,25 @@ slices:
       ]
       content.write("/etc/ssl/certs/ca-certificates.crt", "".join(certs))
 
+  default-certs:
```

Comment:
This is confusing to me.

Why do we need this slice? Whoever is using `_data` will continue with a broken installation. That slice itself, is broken. If you want an additional slice for those who don't need the bundled cert file, then ok, we can have an additional one that behaves like `_data` today, but `_data` is the one that needs fixing